fastlane/lib/fastlane/helper/crashlytics_helper.rb

```diff
@@ -57,12 +57,6 @@ def generate_ios_command(params)
         end
 
         def generate_android_command(params)
-          # We have to generate an empty XML file to make the crashlytics CLI happy :)
```

Comment:
unrelated deletion?

Reply:
Moved to https://github.com/fastlane/fastlane/pull/14037/files#diff-23c7195bf7a4d9b35a0a2076e247b544R30